pokemon/src/components/App/index.tsx

```diff
@@ -0,0 +1,37 @@
+import DetailPage from '@components/DetailPage';
+import List from '@components/List';
+import { usePokemon } from '@contexts/Pokemon';
+import { useEffect } from 'react';
+
+function App() {
+  const { list, fetchNextPage, setSelectedName } = usePokemon();
+
+  useEffect(() => {
```

Comment:
useEffect - is this mostly placeholder for an idea at implementation? if you had more time to implement, what's the idea for how fetchNextPage() was supposed to be called?

Reply:
Yeah, it's a placeholder. I considered doing an infinite scroller on the list or adding a "Load more" button, but decided not to do it since there was no design specified and the list was hard-set on using the first 151 Pokémon. I'd probably go for an infinite scroll here in a normal scenario.